### PR TITLE
Add privacy notice page

### PR DESCRIPTION
**Pull request by Renny4Real**
**IN THIS PR:**
- Add privacy notice page & [copied content](https://www.gov.uk/government/publications/govwifi-privacy-notice/govwifi-privacy-notice) from GOV.UK page.

Screenshot 2019-04-25 at 15 31 04

------------------------------------------------------------------------------------------------------

- Add anchor links and side bar navigation for the page.

- Add sticky styling to column to mimic [GOV.UK Registers'](https://www.registers.service.gov.uk/privacy-notice) version of navigation on that page.
(This was requested.)

Screenshot 2019-04-25 at 15 31 22

_Notice that the sidebar sticks to the screen when it's position reaches the top, even though the rest of the page is mid scroll_ 

------------------------------------------------------------------------------------------------------

**Any feedback/suggestions would be appreciated.**

### CODE REVIEW

File: source/stylesheets/modules/_sticky.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+//
+// Example Usage:
+//
+// <span class="header__title">
+//   My Product <span class="sticky">Beta</span>
+// </span>
+
+.sticky {
+  position: -webkit-sticky;
+  position: sticky;
```
Comment (emileswarts) [inline source/stylesheets/modules/_sticky.scss]: Can you have two properties on one class? I thought they would have overwritten each other.

File: source/shared/_privacy_notice_sidebar.html.erb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  { title: "Where it might go", link: "#where-it-might-go" },
+  { title: "What are your rights", link: "#what-are-your-rights" },
+  { title: "Changes to this notice", link: "#changes-to-this-notice" },
+  { title: "How to contact us", link: "#how-to-contact-us" }
+] %>
+
+<nav class="sub-navigation">
+  <ul>
+    <% menu_items.each do |menu_item| %>
+      <li class="sub-navigation__item <%= subnav_active(menu_item.fetch(:link)) %>">
```
Comment (emileswarts) [inline source/shared/_privacy_notice_sidebar.html.erb]: Is this active state functionality going to work with anchor tags?

Comment (Renny4Real) [reply source/shared/_privacy_notice_sidebar.html.erb]: I've tried it, it doesn't but Register's version also doesn't have an active state. 
Wasn't sure if there was a way to make it work. 
I also could remove this.

File: source/stylesheets/modules/_sticky.scss (hunk 1 above)
Comment (Renny4Real) [reply source/stylesheets/modules/_sticky.scss]: I think it depends on the browser. 
Safari requires the `position: -webkit-sticky`.

Comment (emileswarts) [reply source/stylesheets/modules/_sticky.scss]: Ah ok

**Pull request merged**